EvalView/templates/EvalView/data-assessment.html

```diff
@@ -184,12 +184,12 @@
     <div class="item-box item-{% cycle 'odd' 'even' %} quotelike">
         <div class="row">
             <div class="col-sm-6">
-                <span title="Source sentence #{{ loop.index|add:"1" }}">
+                <span title="Source sentence #{{ forloop.counter }}">
                     <p>{{ source_text|safe }}</p>
```

Comment:
This is the line (and corresponding one below) where we would want to replace `|safe` with `|escape`?